README.md

```diff
@@ -83,6 +84,7 @@ If all goes well something like the following output will be displayed:
 ```
 
 You should now be able to view a live preview at http://localhost:4567.
+In addtioned, this links above will be accessible on local host url when you run the boundle server. 
```

Comment:
3 typos on this line
addtioned, local host, boundle

I'm unsure what this line is even adding on top of the line above